app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
+end
```

Comment:
- [ ] The `belong_to` should belong to the author and the author is associated with the User table cause in the table we have `author_id` not `user_id`, it can be similar to 👉🏾 `belongs_to :author, class_name: 'User'`. Kindly fix that 😁
- [OTIPNAL] Same remark about using the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) for the `update_comments_counter`.
The code can be similar to 👇 

```ruby
  after_save :update_likes_counter

  private

  def update_likes_counter
    author.increment!(:likes_counter)
  end
```